scripts/e2e-tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+TEST_NAMESPACE="openshift-marketplace"
+MANIFEST_FOLDER="./test/e2e/environment/"
+NAMESPACED_MANIFEST="./${MANIFEST_FOLDER}/namespaced-manifest.yaml"
+GLOBAL_MANIFEST="./${MANIFEST_FOLDER}/global-manifest.yaml"
+OPERATOR_SOURCE_CRD="./deploy/crds/operatorsource.crd.yaml"
+CATALOG_SOURCE_CONFIG_CRD="./deploy/crds/catalogsourceconfig.crd.yaml"
+
+# Create openshift resources if they don't exist
+echo "Creating openshift resources"
+kubectl apply -f $OPERATOR_SOURCE_CRD
+kubectl apply -f $CATALOG_SOURCE_CONFIG_CRD
+if ! kubectl get namespace $TEST_NAMESPACE; then
+    kubectl create namespace $TEST_NAMESPACE
+fi
+
+# Run the tests through the operator-sdk
+echo "Running operator-sdk test"
+operator-sdk test local ./test/e2e --up-local --kubeconfig=$KUBECONFIG --namespace $TEST_NAMESPACE
```

Comment:
Yes, exactly that. That way, we can run the operator from source rather than relying on an image built from an external process. Without that tag the test relies on the image referenced in the operator.yaml

Reply:
Okay, so due to tools limitation we can't deploy marketplace operator on the cluster using the individual yaml files and then run the `e2e` tests on it. 

(In the future) I believe e2e tests should run on an environment provisioned from the deploy manifests.  For example, (thinking out loud here) we could templatize `namespace` and `image` referred to by individual yaml files. We build the image, generate the yaml files from templates, deploy and run tests.I am sure there are more elegant solutions we can come up with in the future. It would be great if we track this issue in jira.

Reply:
A lot of what you are asking for is done by the CI operator once Marketplace is part of the cluster payload image-stream and will run by default in a cluster.